frozen/pewpew10/pew.py

```diff
@@ -0,0 +1,217 @@
+from micropython import const
```

Comment:
I'd suggest not including this Python code in the `adafruit/circuitpython` repo, but instead in a separate submodule repo as a library like we do the other frozen modules. Then it can be modified experimentally by changing the library repo code and changing `sys.path. If it needs to be updated, it's a very simple PR to just update the submodule commit.

Reply:
Can this repository live outside of adafruit's github?

Reply:
Sure, we depend on plenty of submodule repos that are outside the adafruit organization (though no libraries yet).